vufs/src/main/java/com/stnetix/ariaddna/vufs/BusinessObjects/MasterTableType.java

```diff
@@ -0,0 +1,8 @@
+package com.stnetix.ariaddna.vufs.BusinessObjects;
+
+/**
+ * Flag of type metatable
+ */
+public enum MasterTableType {
+        MASTER, SNAPSHOT
```

Comment:
for any enum please add toString method. It's make own logs more simple for read.